docs/api-v2-import.json

```diff
@@ -0,0 +1,520 @@
+            "explode": true,
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/components/schemas/EventImportStatus"
+              }
+            },
+            "description": "An optional filter to specify the status of the event import.\n"
+          },
+          {
+            "name": "after",
+            "in": "query",
+            "schema": {
+              "type": "string",
+              "x-tie-haskell-type": "Data.UUID.UUID"
+            },
+            "description": "A cursor that is used for pagination.\n"
+          },
+          {
+            "name": "per_page",
+            "in": "query",
+            "required": false,
+            "schema": {
+              "type": "integer",
+              "example": 30
+            },
+            "description": "A cursor used for pagination\n"
+          }
+        ],
+        "get": {
+          "operationId": "getEventImports",
+          "summary": "Retrieve a list of event imports\n",
+          "responses": {
+            "200": {
+              "description": "A list of event imports",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "$ref": "#/components/schemas/EventImports"
+                  }
+                }
+              }
+            },
+            "403": {
+              "$ref": "#/components/responses/Unauthorized"
+            },
+            "404": {
+              "$ref": "#/components/responses/NotFound"
+            }
+          }
+        }
+      },
+      "/v2/imports/{owner}/{ticket}": {
+        "parameters": [
+          {
+            "name": "owner",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "Owner of the event import.\n"
+          },
+          {
+            "name": "ticket",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "x-tie-haskell-type": "Data.UUID.UUID"
+            },
+            "description": "Id of the event import to retrieve.\n"
+          }
+        ],
+        "get": {
+          "operationId": "getEventImport",
+          "summary": "Retrieve a specific event import\n",
+          "responses": {
+            "200": {
+              "description": "Returns the EventImport specified by the ticket",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "$ref": "#/components/schemas/EventImport"
+                  }
+                }
+              }
+            },
+            "403": {
+              "$ref": "#/components/responses/Unauthorized"
+            },
+            "404": {
+              "$ref": "#/components/responses/NotFound"
+            }
+          }
+        }
+      },
+      "/v2/imports/{owner}/{ticket}/abort": {
```

Comment:
Why would one want to abort an import? I think some explanation here will be useful.

Reply:
> Why would one want to abort an import? I think some explanation here will be useful.

I did add something to the overview. In short, importing events is a costly + asynchronous operation, being able to abort it for whatever reason is something we should be able to do at any time.